README.md

```diff
@@ -248,5 +248,51 @@ else
 end
 ```
 
+### End-to-end encryption
+
+This library supports end-to-end encryption of your private channels. This means that only you and your connected clients will be able to read your messages. Pusher cannot decrypt them. You can enable this feature by following these steps:
```

Comment:
I think it would be good to include a link to this page: https://pusher.com/docs/channels/using_channels/encrypted-channels

Reply:
good shout